src/styles/styles.js

```diff
@@ -2198,12 +2198,12 @@ const styles = {
         flexDirection: 'row',
         flexWrap: 'wrap',
         gap: 12,
-        height: 148,
+        minHeight: 148,
```

Comment:
Question, why do we need a minHeight or height? Is it possible to let the container have the height defined by the content?

Reply:
Hmm yeah, I thought this is made to make it look nice when no code is loaded, i.e. loading the spinner but the spinner uses it own View with a fixed height already.

Reply:
ahh maybe it is to try to keep the box the same height when the loading spinner switches to the actual codes.

Reply:
so maybe we keep `minHeight`... does the use of `minHeight` instead of `height` fixes this: https://github.com/Expensify/App/pull/21698#issuecomment-1613521538?

Reply:
I see, we haven't seen any bad effect of using `height`, right? If we have not, maybe we should test with that? unless we have some strong reason not to use it.

Reply:
The use of height does not work well for small screens and may lead to bugs like this https://github.com/Expensify/App/pull/19260#issuecomment-1567314575.  The correct use is `minHeight` but I think there is a bug in RN, it applies the min height for every row making it taller than it should.

i think we can just remove height/minHeight as the spinner has it's height set so it will still look correct in case no code is loaded.

Reply:
No we should only show the spinner if we are actually loading something. But in what cases no code is loaded (and the spinner is not loading either)

Reply:
> But in what cases no code is loaded (and the spinner is not loading either)

No cases? 🤷 

> So we only need to remove minHeight and height. @aldo-expensify What do you think about this, if we come to the conclusion I will update the PR right away for this.

I guess that sounds fine. Sounds good to you @s77rt ?

Reply:
Yes sounds good to me 👍